doc/sphinx/methods.rst

```diff
@@ -450,6 +450,33 @@ After the calculate() method is called, the following attributes are available:
 
 .. index::
     single: methods; Accessing additional methods through bridge
+
+
+Nuclear Properties
+------------------
+
+This method calculates various properties of nuclei based on the parsed data.
+
+Calculate the nuclear repulsion energy using the snippet below:
+
+.. code-block:: python
+    import cclib
+    from cclib.method import Nuclear
+
+    parser = cclib.io.ccopen("test.log")
+    data = parser.parse()
+    nuclear = Nuclear(data)
+
+    nre = nuclear.repulsion_energy()
+    print(nre)
+
+The value can be also converted to Hartrees (to directly compare with Gaussian logfiles):
```

Comment:
You've exposed a bug.  All energies that aren't excitation energies are supposed to be in eV (yeah, I know...), but your conversion snippet below does indeed produce the number in Hartrees.  That makes the result sort of doubly wrong.

I'd rather fix the units on the return value before updating these docs.  One way around that though is to fix the unit bug separately after merging this PR with the `convertor(...` snippet removed, and one can safely assume it has the same units as all other energies that are documented.

What do you think?

Reply:
This sounds great.